index.rst

```diff
@@ -1,59 +1,367 @@
+By analogy with raw and calibration data, these will be stored in a ``HSC/masks/S18A`` ``RUN`` collection, with a ``HSC/masks`` single-element ``CHAINED`` collection pointer to the current best version.
+``S18A`` refers to the HSC internal release in which these masks were first used.
+While it is somewhat unlikely that we will ever add older mask versions or new masks in the same form to LSST data repositories (LSST processing is moving to a different approach to these masks, and HSC will probably follow suit), this gives us a clear place to put them without naming conflicts.
+The top-level ``HSC/defaults`` collection will include ``HSC/masks`` as well.
+
+.. _collections-reference-catalogs:
+
+Reference catalogs
+------------------
+
+External reference catalogs reformatted and sharded by DM code are written to ``refcats/<ticket>`` ``RUN`` collections, where ``<ticket>`` is the ticket on which the reformatting and sharding work was performed.
+After a reference catalog has been validated, its ``RUN`` is added to the overall ``refcats`` ``CHAINED`` collection.
+
+Different collections for different reference catalogs are not necessary, as the name of a reference catalog (e.g. ``ps1_pv3_3pi_20170110``) is used directly as its dataset type name (note that this was not the case in Gen2, where the reference catalog name was instead part of the data ID).
+
+.. _collections-skymap-definitions:
+
+SkyMap definitions
+------------------
+
+All skymaps must have a globally unique name in Gen3, which is used as part of the data ID for any dataset that is defined on tracts.
+The skymap definition datasets (i.e. ``lsst.skymap.BaseSkyMap`` subclass instances in Python) also include this globally unique name in their data IDs, and hence can also all go in a single ``skymaps`` collection.
+This is simply a ``RUN`` collection that holds skymap definition datasets directly.
+
+The existence of different skymap definition datasets for different coadd types (``goodSeeingCoadd_skyMap``, etc.) is a relic of Gen2 that will soon be removed entirely from Gen3; all skymap definition datasets will just use the ``skyMap`` dataset type.
+The new globally-unique skymap data ID names are both necessary and sufficient for uniqueness in Gen3.
+
+SkyMap registration is something we expect to be rare in Gen3 - *much* more rare than running ``makeSkyMap.py`` was in Gen2 - because we almost always use one of a few standard SkyMaps, and in Gen3 a SkyMap (a combination of a ``lsst.skymap.BaseSkyMap`` class *and* its configuration) can only be registered once.
+Discrete SkyMaps, which typically cover only a small part of the sky and are *conceptually* a bit more per-user, maybe less rare, but our data model currently does not treat these any differently, and until we can identify the patterns and use cases for creating new SkyMaps (even discrete ones), we propose that any new SkyMap registration in a shared repository be preceded by an RFC.
```

Comment:
I'm using "propose" in this document only because the RFC is still open. Some time between adopting and implementing it the language will change, possibly only when it is moved to other, more canonical documents.

I would like to give developers the ability to temporarily assume write access even for things that require an RFC; I think there's a fairly large efficiency gap between "adopt your RFC (yourself), add the skymap (yourself), mark the RFC as implemented (yourself)", and something that involves a sysadmin doing a chmod or something for you in between, and little to be gained from the latter.

Reply:
"temporarily assume" - Ok, I think that satisfies my concern. I'm more worried about "oops, I didn't mean to do that" than anything malicious, and requiring explicit adopting of a temporary role via `su` or equivalent mostly takes care of that.